### ISSUE TITLE
[discussion] Future of org-bullets

### ISSUE THREAD

**Issue by integral-dw** [syl20bnr/spacemacs#13831]
Hello!  While I am not an active user of spacemacs, [it has come to my attention](https://develop.spacemacs.org/layers/%2Bemacs/org/README.html) that it ships with sabof's [org-bullets](https://github.com/sabof/org-bullets).  I am the current maintainer of the [MELPA fork](https://github.com/integral-dw/org-bullets), which has diverged from the main code with a variety of patches in the past 6 years.  For that reason (and to avoid a duplication of effort) I've come here primarily to ask whether you would like to use my fork of the package instead, which is my first step towards trying to unify the user experience of the package across Emacs distributions.  But why do I say "_primarily_"?  Well... I think I should give a bit of context for that.

The package `org-bullets`, while immensely popular with [over 600k downloads on MELPA](https://melpa.org/#/org-bullets), has matured enough that I would personally consider any major changes to it hard to justify towards the end-user.  This, combined with the fact that to my knowledge several versions of this package already exist, led me to the decision that it would be best to "freeze" the package, only applying bug fixes and general maintenance to keep it running, not create further divergence among packages of the same name.
However, that begs the question: what to do with [existing feature requests](https://github.com/sabof/org-bullets/issues) that never ended up in the package? What with features I myself would have liked to see?  Enter [org-superstar](https://github.com/integral-dw/org-superstar-mode), a package I have written from scratch after a thorough dissection of org-bullets.  While it has taken ideas from org-bullets, it shares no code with it, and extends the ideas of org-bullets, and can emulate its function, making it more of a spiritual successor than a fork.  Consequently, it is still very much active and subject to change (although it is mature enough that I will not break backwards compatibility with older versions of itself without warning).  Superstar seems to do quite well, too: it already reached [over 17k downloads on MELPA](https://melpa.org/#/org-superstar) in roughly half a year since its inception.
Which finally brings me to my second suggestion: Would you be interested in replacing org-bullets with org-superstar?  

So to summarize, would you be interested in
- changing your upstream repo for `org-bullets` to the MELPA one for a more unified user experience, or
- replacing the package with a successor package (`org-superstar`) offering some additional bells and whistles?

I'll gladly go into more detail about the differences between the two packages, if need be.

**Comment by jr0cket** [comment]
The `develop` branch of Spacemacs has already been changed over to `org-superstar`, via a commit merged on 7th June 2020

**Comment by integral-dw** [comment]
Ah, awesome!  :partying_face:   I tried to make the transition from Bullets to Superstar as painless as possible, but should you come across something bothersome I have overlooked be sure to let me know.
